ProcessMaker/Http/Controllers/Api/ExportController.php

```diff
@@ -19,6 +18,10 @@
 use ProcessMaker\Models\ProcessTemplates;
 use ProcessMaker\Models\Screen;
 use ProcessMaker\Models\Script;
+use ProcessMaker\Package\PackageDecisionEngine\ImportExport\DecisionTableExporter;
+use ProcessMaker\Package\PackageDecisionEngine\Models\DecisionTable;
+use ProcessMaker\Packages\Connectors\DataSources\ImportExport\DataSourceExporter;
+use ProcessMaker\Packages\Connectors\DataSources\Models\DataSource;
```

Comment:
Hey @estebangallego before initializing these classes, could we implement a check to verify if the corresponding packages are installed or if the classes exist? This precautionary step would help avoid potential issues for instances that do not have these packages installed or for newly spun-up instances. 

So for example:

```
use ProcessMaker\PackageHelper;

const DATA_SOURCE_CLASS = 'ProcessMaker\Packages\Connectors\DataSources\Models\DataSource';
const DATA_SOURCE_EXPORTER_CLASS = 'ProcessMaker\Packages\Connectors\DataSources\ImportExport\DataSourceExporter';

const DECISION_TABLE_CLASS = 'ProcessMaker\Package\PackageDecisionEngine\Models\DecisionTable';
const DECISION_TABLE_EXPORTER_CLASS = 'ProcessMaker\Package\PackageDecisionEngine\ImportExport\DecisionTableExporter';

const IS_DATA_SOURCES_INSTALLED = PackageHelper::isPackageInstalled(self::DATA_SOURCE_CLASS);
const IS_DECISION_TABLE_INSTALLED = PackageHelper::isPackageInstalled(self::DECISION_TABLE_CLASS);

class ExportController extends Controller
{
    protected array $types = [
        'process' => [Process::class, ProcessExporter::class],
        'script' => [Script::class, ScriptExporter::class],
        'process_templates' => [ProcessTemplates::class, TemplateExporter::class],
        'decisiontable' => IS_DECISION_TABLE_INSTALLED ? [self::DECISION_TABLE_CLASS, self::DECISION_TABLE_EXPORTER_CLASS] : null,
        'datasource' => IS_DATA_SOURCES_INSTALLED ? [self::DATA_SOURCE_CLASS, self::DATA_SOURCE_EXPORTER_CLASS] : null,
    ];
}
```
Let me know if you have questions or run into problems with the above code